modules/client/src/test/java/org/apache/ignite/client/AbstractClientTableTest.java

```diff
@@ -119,7 +119,9 @@ protected static Tuple allColumnsTableVal(String name) {
     }
 
     protected Table fullTable() {
-        server.tables().createTableIfNotExists(TABLE_ALL_COLUMNS, tbl -> tbl.changeReplicas(1));
+        if (server.tables().table(TABLE_ALL_COLUMNS) == null) {
+            server.tables().createTable(TABLE_ALL_COLUMNS, tbl -> tbl.changeReplicas(1));
```

Comment:
Seems that option with ignoring TableAlreadyExists will be a bit faster than explicit `if (server.tables().table(TABLE_ALL_COLUMNS) == null)` , what do you think?

Reply:
It would be a logic on exception.